src/people/widgetViews/__tests__/OrgHeader.spec.tsx

```diff
@@ -90,11 +105,4 @@ describe('OrgHeader Component', () => {
     expect(websiteButton).toBeInTheDocument();
     expect(githubButton).toBeInTheDocument();
   });
-
-  it('UrlButtons are left-aligned if visible', () => {
-    const { getByTestId } = render(<OrgHeader {...MockProps} />);
-    const urlButtonContainer = getByTestId('url-button-container');
-    const containerStyle = window.getComputedStyle(urlButtonContainer);
-    expect(containerStyle.marginLeft).toBe('0px');
-  });
```

Comment:
is there a test to still assert that `url-button-container` exists we shouldn't remove tests and not add new ones in its place

Reply:
The reason is that url button is now displayed conditionally and if the link is not in the url button will not be displayed on the page